apps/web/src/pages/Pools/IndexPoolPage/components/PoolTable.tsx

```diff
@@ -276,11 +324,14 @@ export const PoolTable = ({ pools }: PoolTableProps) => {
           id: "tpzt/B",
           description: "Title displayed for the Pool table Edit column.",
         }),
-        accessor: (d) =>
+        accessor: () => {
+          return "Edit";
+        },
```

Comment:
I think if we are just going to return the same value for all accessors on this column it would be best to disable sorting on this column. I would also disable search on this column as well since when I search for "Edi" all rows show up since they technically all satisfy that search string.

Reply:
oh and this one https://github.com/GCTC-NTGC/gc-digital-talent/pull/5631/commits/e24b2faf554313e3d99be15bedb48b078532c32e